tensorflow/core/ops/ops.pbtxt

```diff
@@ -13604,6 +13604,74 @@ op {
     minimum: 1
   }
 }
+op {
+  name: "LeakyRelu"
+  input_arg {
+    name: "features"
+    type_attr: "T"
+  }
+  output_arg {
+    name: "activations"
+    type_attr: "T"
+  }
+  attr {
+    name: "alpha"
+    type: "float"
+    default_value {
+      f: 0.2
+    }
+  }
+  attr {
+    name: "T"
+    type: "type"
+    default_value {
+      type: DT_FLOAT
+    }
+    allowed_values {
+      list {
+        type: DT_HALF
+        type: DT_FLOAT
+        type: DT_DOUBLE
+      }
+    }
+  }
+}
+op {
+  name: "LeakykReluGrad"
```

Comment:
I don't believe these changes are required (they are auto generated), but in case they are, you have a typo here - LeakykReluGrad -> LeakyReluGrad.

Reply:
Really a typo. Thanks for figuring it out. :)